src/agent/src/mount.rs

```diff
@@ -622,8 +622,7 @@ pub fn get_mount_fs_type_from_file(mount_file: &str, mount_point: &str) -> Resul
     let file = File::open(mount_file)?;
     let reader = BufReader::new(file);
 
-    let re = Regex::new(format!("device .+ mounted on {} with fstype (.+)", mount_point).as_str())
-        .unwrap();
+    let re = Regex::new(format!("device .+ mounted on {} with fstype (.+)", mount_point).as_str())?;
```

Comment:
As with the other use of Regex, a failure here indicates a bug in the code, not a problem in the inputs, which makes `unwrap()` appropriate.